README.md

```diff
@@ -1,56 +1,72 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Budget App
 
-> Description the project.
+>  a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
+>  Ruby on Rails application that allows the user to:
+>  - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
+## Design 
+- #### Original design idea by: [Gregoire Vella on Behance.](https://www.behance.net/gregoirevella)
+- #### Design [link.](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Languages Used: HTML, CSS, RUBY.
+- Frameworks: RUBY ON RAILS, BOOTSTRAP.
+- Technologies used: RSpec, Capybara, Bullet, Devise, CanCanCan.
 
-## Live Demo (if available)
+## Live Demo (Soon)
 
-[Live Demo Link](https://livedemo.com)
+[Not Available](https://livedemo.com)
 
+## Video (Soon)
```

Comment:
- [ ] Let's add valid video and live version of you application.
Remember to mention a clear and objective description about the project. Also highlight a piece of code that you find interesting or you are proud of making.
- [ ] command to run you test should be `rspec spec` instead of `rspec test`. Just confirm.